README.md

```diff
@@ -1 +1,61 @@
-# framework-awesomebook
\ No newline at end of file
+# framework-awesomebook
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# awesomebook
+
+> This a group work project testing basic Javascript skills.
+
+# WORKFLOW SCREENSHOT
+![screenshot]()
+
+## Built With
+- HTML
+- CSS
+- JavaScript
+## Live Demo (if available)
+
+[Live Demo Link](https://siansandatamara.github.io/framework-awesomebook/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Web Browser
+- Code Editor
+- Prior knowledge of CSS, HTML, GIT, GITHUB and Linters.
+
+### Setup
+
+```bash
+git clone// https://siansandatamara.github.io/awesome-book/
+cd awsomebook
+
```

Comment:
### **OPTIONAL**
- [ ] Kindly replace the below code to solve the problem with your readme file. You opened a special character to write a block of code and you didn't close it for that reason any other text below it is affected.
![bash](https://user-images.githubusercontent.com/55596970/164393587-d40e3b32-6741-4654-8843-0b57bb6aaa6d.png)